### ISSUE TITLE
[Crash] [1.16 Pre 2] Game crashes when killing lots of entities

### ISSUE THREAD

**Issue by ewanhowell5195** [sp614x/optifine#4240]
## Description of Issue
The game will crash when you run a `/kill @e` command and there are a lot of entities

## Steps to Reproduce
Run `/kill @e` when there are a lot of mobs around
Alternatively download this world ive been testing in and use the command blocks here
[Mob Test World.zip](https://github.com/sp614x/optifine/files/4873547/Mob.Test.zip)

## OptiFine Version
1.16.1 Pre 2

## Installation Method
Installer

## Log Files/Crash Reports
[Throws an `Unknown Error` crash](https://pastebin.com/wuqH0JYP)

To open this menu, simply press `F3` on your keyboard while in-game. Please ensure the text is easily readable. Depending on the size/resolution of your monitor, you may have to adjust your GUI scale for all text to be completely visible on-screen.

## Prior Testing
Only happens with OptiFine. Tested Vanilla 1.16.1

## F3 Screen
![image](https://user-images.githubusercontent.com/48070995/86518387-aa667780-be29-11ea-80e3-ee345acc3ffa.png)

## Videos

Vanilla 1.16.1:
https://imgur.com/a/dE4rg9P

OptiFine 1.16.1 Pre 2:
https://imgur.com/a/jFB3ifY

**Comment by XxZajoZzO** [comment]
Its not a lot of mobs that are crashing this. Its the pahntom. Kill the pahntom with the sword and you will see it will crash.

**Comment by ewanhowell5195** [comment]
no it was crashing before i even added the phantom to the world

**Comment by ewanhowell5195** [comment]
there might be other mob killing crashing issues, as it did crash on me multiple times before when killing small groups of entities, but doing them all at once makes it a reliable crash

**Comment by XxZajoZzO** [comment]
Strange stuff. I am killing them one by one randomy and sometomes its crashing sometimes not. I also managed to kill a phantom to without crash but /kill crashed.

**Comment by ewanhowell5195** [comment]
are you using the world i linked?

**Comment by XxZajoZzO** [comment]
Yes

**Comment by Shakar001** [comment]
I would like to addon to this that I've noticed this happening with the Parrots specifically. I thought maybe the fabric modded minecraft was the cause, so I tried vanilla. Vanilla minecraft had no issue. I tried Vanilla with just optifine, and then killing parrots (Tamed and wild) caused the game to crash. I'm not sure about other mobs as I haven't crashed with them, but I've been getting the same if not a very similar error when parrots die. To clarify, this is on the same build being reported on in this issue, minus active shaders.
